packages/react-native/Libraries/Utilities/DevMenu.d.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ */
+
+/**
+ * The DevMenu module exposes methods for show the dev menu in development.
+ */
+export interface DevMenuStatic {
+  /**
+   * Show the developer menu.
```

Comment:
Same nit. Plus — can we replicate these doc comments in the Flow source as well?

```suggestion
   * Show the Dev Menu.
```